content/docs/5.1/develop/clients/go.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Go Client
+description: Interact with TiKV using Go.
+menu:
+    "5.1":
+        parent: TiKV Clients
+        weight: 2
+---
+
+TiKV client for go is still in the stage of prove-of-concept and under heavy development. You can track development at [tikv/client-go](https://github.com/tikv/client-go/) repository.
```

Comment:
Seems https://github.com/tikv/client-go/tree/v2 is available.

Reply:
menglong suggested to update this document until v2 is released.